packages/rum/README.md

```diff
@@ -107,19 +149,10 @@ parents) that will be used to name the action. Examples:
 </div>
 ```
 
-## TypeScript support
-
-Types are compatible with TypeScript >= 3.0.
-For earlier version, you can import js sources and use global variable to avoid any compilation issue:
-
-```
-import '@datadog/browser-rum/bundle/datadog-rum';
-
-window.DD_RUM.init({
-  applicationId: 'XXX',
-  clientToken: 'XXX',
-  site: 'datadoghq.com',
-  resourceSampleRate: 100,
-  sampleRate: 100
-});
-```
+[1]: https://app.datadoghq.com/rum/list
+[2]: /real_user_monitoring/data_collected/
+[3]: /real_user_monitoring/dashboards/
+[4]: https://www.npmjs.com/package/@datadog/browser-rum
+[5]: /account_management/api-app-keys/#client-tokens
+[6]: /real_user_monitoring/data_collected/user_action/#automatic-collection-of-user-actions
+[7]: /real_user_monitoring/faq/proxy_rum_data/
```

Comment:
relative links won't work from github, could we use full url instead?
ex: `/real_user_monitoring/data_collected/` -> `https://docs.datadoghq.com/real_user_monitoring/data_collected/`